docs/tutorials/05_qaoa.ipynb

```diff
@@ -0,0 +1,392 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quantum Approximate Optimization Algorithm\n",
+    "\n",
+    "Qiskit has an implementation of the Quantum Approximate Optimization Algorithm [QAOA](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.QAOA.html) and this notebook demonstrates using it for a graph partition problem.\n",
+    "\n",
+    "Before we begin, let's import the `annotations` module from `__future__` to allow postponed evaluation of annotations. This enables us to use simpler type hints throughout the notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from __future__ import annotations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First we create a graph and draw it so it can be seen."
+   ]
```

Comment:
> draw it so it can be seen

"it" is not explained at all. Perhaps this should more say we are going to solve a maxcut graph problem where we define the graph via an adjacency matrix and to visualize the problem we draw the graph....